Add GitHub issue and pull request templates (#47)
PSPDFKit/pspdfkit-flutter: .github/PULL_REQUEST_TEMPLATE.md

# Details

# Acceptance Criteria

> - [ ] When approved, right before merging, make sure that all examples work on both Android and iOS.

Reviewer: ```suggestion
- [ ] Before approving, make sure the examples work on both Android and iOS.
```

Reviewer 2: Hmm. I would say that the person approving the PR needs to ensure nothing is broken. Ultimately, we should have automated tests for this. Until then, the owner of the PR should do the check.

Reviewer: I have explained this very badly. 
What I wanted to envisage is: before making a PR ready to review -> check the examples

Author: I agree with @davidschreiber! Let's leave it as-is for now: The owner of the PR should test after the PR has been approved and right before merging.

Reviewer: I don't see the point in doing a manual check _after_ the approval.
If something is broken better to know it in advance rather later.

What if something does not work and it has been approved? 
(It needs to be fixed and then trigger again another review?)

We should ask for a review only if we are sure an implementation works.
